.github/workflows/test-ga.yml

```diff
@@ -44,3 +45,19 @@ jobs:
       - name: Cleanup
         run: make cb-stop
         working-directory: ${{ env.working-directory }}
+
+      - name: Notify slack on failure
+        # We run this step only on the default 3.1+ branch
```

Comment:
commented version is wrong for Go?

Reply:
ah! totally missed, I will update in here and in 2.3 (damn versioning confusion...)